src/main/java/org/concordion/integration/junit4/ConcordionRunner.java

```diff
@@ -168,7 +202,7 @@ void invoke(ConcordionFrameworkMethod concordionFrameworkMethod) {
             // to worry about them. But it doesn't invoke the @Before and @After methods. So we explicitly
             // invoke them here.
 
-            ResultSummary result = fixtureRunner.run(example);
+            ResultSummary result = fixtureRunner.run(example, fixture);
```

Comment:
We're not calling Before and AfterSpecification in this case. As far as I can see this path is only taken by JUnit 3, so I'm OK with that. I think we should document that the new functionality is not supported with JUnit3 though.

We've deprecated JUnit 3, so maybe we should deprecate this whole method?